tests/client_fixture.py

```diff
@@ -13,6 +13,18 @@ def client():
     return TestClient(app)
 
 
+@pytest.fixture(scope="session")
```

Comment:
How about using the already exists TestClient? :)

Reply:
I saw that each test client changes only the get_db function for the module it's testing, not sure why really but I thought I'd go with it instead of changing other people code.

Reply:
There's no need to, you can use the general TestClient in `conftest.py`